packages/sit-onyx/src/components/OnyxToastMessage/OnyxToastMessage.vue

```diff
@@ -21,7 +21,7 @@ const emit = defineEmits<{
   /**
    * Emitted when the toast is clicked. Requires `clickable` property to be enabled.
    */
-  click: [];
+  toastMessageClick: [];
```

Comment:
Would prefer to still call this click but I guess thats not the intention of the ticket...

Reply:
We should remove the event completely. When it is not clickable and the developer listens for a click event anyways, we won't be able to circumvent that anyways.
Alternatively I would propose a renaming to a more generic verb, maybe `trigger`?